oaknational/Oak-Web-Application
# feat(curric): updating curriculum tab structure, adding data and test

## Description

- Updated routing in curriculum info tabs (overview, units, downloads)
- Renamed `sequence` to `units`
- Added fixtures for missing data
- Updated tab styling to show current tab

## How to test

1. Go to {owa_deployment_url}
2. Click on tab `Unit Sequence` and then click on tab `Downloads`
3. You should see the updated page URL, current tab, and tab content
